applications/app/services/ApplicationsInteractiveRendering.scala

```diff
@@ -29,13 +29,43 @@ object DotcomRendering extends RenderingTier
  */
 
 object ApplicationsInteractiveRendering {
+
+  // allowListedPaths is use to jumpstart the router (which decides which between frontend and DRC does the rendering)
+  val allowListedPaths = List(
+    "/sport/ng-interactive/2018/dec/26/lebron-james-comments-nba-nfl-divide",
+  )
+  def ensureStartingForwardSlash(str: String): String = {
+    if (!str.startsWith("/")) ("/" + str) else str
+  }
+
+  def router(path: String)(implicit request: RequestHeader): RenderingTier = {
+    // This function decides which paths are sent to DCR for rendering
+    // At first we use allowListedPaths
+    if (allowListedPaths.contains(ensureStartingForwardSlash(path))) DotcomRendering else FrontendLegacy
+  }
+
   def getRenderingTier(path: String)(implicit request: RequestHeader): RenderingTier = {
+
     val isSpecialElection = ApplicationsUSElection2020AmpPages.pathIsSpecialHanding(path)
-    val isAmp = request.host.contains("amp")
+
+    // Date   : 01st June 2021
+    // Author : Pascal
+    // Note   : We define isWeb as the opposite of isAmp because I think it leads to easier to understand code
+    //          If it turns out that somebody disagrees, let me know.
+    val isWeb = !request.host.contains("amp")
+
     val forceDCR = request.forceDCR
 
-    if (isSpecialElection && isAmp) USElection2020AmpPage
-    else if (forceDCR) DotcomRendering
-    else FrontendLegacy
+    (isSpecialElection, isWeb, forceDCR) match {
```

Comment:
This kind of match can be a bit tricky to read because you have to map the case placeholders to the initial match line.

It seems easier/more readable to just do:

```
case electiontracker if isSpecialElection && isAmp => USElectionTracker2020AmpPage
case dcr if forceDCR => DotcomRendering
case _ => FrontendLegacy
```

I'm not sure I understand the router(path) case here.

Reply:
Done!